startup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I checked briefly how lib-elf is used and structured, I think we should be able to bring it in exactly like libbpf? Eg, workspace rule, and BUILD rule.

The bpf-lib target can then have a deps = ["libelf"], for linking, and we can add -I for includes?